src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -80,7 +80,7 @@ protected override (string displayText, string suffix, string insertionText) Get
 
             if (symbol is ITypeSymbol typeSymbol)
             {
-                return base.GetDisplayAndSuffixAndInsertionText(typeSymbol.WithNullability(NullableAnnotation.None), context);
+                return base.GetDisplayAndSuffixAndInsertionText(typeSymbol.WithNullableAnnotation(NullableAnnotation.None), context);
```

Comment:
💡 It would be nice to have a comment explaining this explicit drop of nullability.

Reply:
perhaps this should be changed to "NotAnnotated" as well? Since for object creation we will never annotate the type after a `new`